Tests/NumLib/TestCoordinatesMapping.cpp

```diff
@@ -278,4 +282,38 @@ TYPED_TEST(NumLibFemNaturalCoordinatesMappingTest, CheckZeroVolume)
     ASSERT_ARRAY_NEAR(exp_dNdx, shape.dNdx.data(), shape.dNdx.size(), this->eps);
 }
 
-
+TEST(NumLib, FemNaturalCoordinatesMappingLineY)
+{
+#ifdef OGS_USE_EIGEN
+	typedef Eigen::Matrix<double, 2, 1> NodalVector;
+	typedef Eigen::Matrix<double, 1, 2, Eigen::RowMajor> DimNodalMatrix;
+	typedef Eigen::Matrix<double, 1, 1, Eigen::RowMajor> DimMatrix;
+	typedef Eigen::Matrix<double, 2, 2, Eigen::RowMajor> GlobalDimNodalMatrix;
+#endif
+	// Shape data type
+	typedef ShapeMatrices<NodalVector,DimNodalMatrix,DimMatrix,GlobalDimNodalMatrix> ShapeMatricesType;
+	typedef NaturalCoordinatesMapping<MeshLib::Line, ShapeLine2, ShapeMatricesType> MappingType;
+	double r[] = {0.5};
+	auto line = TestLine2::createY();
+	//std::cout << *line;
```

Comment:
I guess this line and line 303 are debug outputs? Is it possible to remove them or do we need the output later on?